scripts/steamed_hams.py

```diff
@@ -180,13 +178,15 @@ def steamed_hams_parser():
            "radar_control": "",
            "data_write": "",
            "realtime": "",
-           "rx_signal_processing": ""}
+           "rx_signal_processing": "",
```

Comment:
I think this is a great time to take the opportunity to convert this from a dictionary to a list. We only use the keys anyways.

Reply:
It sets up as a dict with keys as module names and empty str values. We then loop to set to start commands as str in the values which is then passed to the screen rc file.